devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -235,8 +235,15 @@ private getModelNumberOfButtons() {[
 
 private getModelBindings(model) {
 	def bindings = []
+	def index = 1
 	for(def endpoint : 1..modelNumberOfButtons[model]) {
 		bindings += zigbee.addBinding(zigbee.ONOFF_CLUSTER, ["destEndpoint" : endpoint])
+		if (isAduroSmartRemote()) {
+			if ((index==2) || (index==3)){
```

Comment:
Do you mean to reference `endpoint` here? `index` is never incremented

Reply:
@tpmanley, Thank you review, i changed the index increment method.

Reply:
@adurosmart I think it would be better to avoid the introduction of an additional `index` variable and just use `endpoint`.  Alternatively, you could leave the `for` loop untouched and put the following after the `for` loop:

```
if (isAduroSmartRemote()) {
    bindings += zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER, ["destEndpoint" : 2]) + zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER, ["destEndpoint" : 3])
```